vars_base

```diff
@@ -5,3 +5,8 @@ export DOCKER_TAG=0.1
 export TF_VAR_mlflow_docker_image=$DOCKER_REPO/$TF_VAR_project_id/$DOCKER_NAME:$DOCKER_TAG
 nb_app=$(gcloud --format json --project $TF_VAR_project_id app services list 2> /dev/null | jq length)
 export TF_VAR_nb_app_engine_services=${nb_app:-0}
+if gcloud app describe 1> /dev/null 2> /dev/null ; then
```

Comment:
Not sure to understand this app_exists, what is the difference with the number of version ? 
If number of version > 0, app exists because an application can not be deleted, or am I missing something ?

Reply:
`app_exists` will be true if the `google_app_engine_application` resource exists, which is desirable since this is the ressource we want to import.

The previous logic was checking if a `google_app_engine_flexible_app_version` existed. This would cause issues if `google_app_engine_application` exists, but no `google_app_engine_flexible_app_version` is deployed

Reply:
Crystal clear, thanks. Might be great to explain it in comment, otherwise could be hard to understand why you did this in a few months.